lib/GridItem.jsx

```diff
@@ -415,6 +458,106 @@ export default class GridItem extends React.Component<Props, State> {
     );
   }
 
+  delayEvents = React.createRef();
+  addEvent = (type, event, passive = true) => {
+    if (this.elementRef.current) {
+      this.elementRef.current.addEventListener(type, event, {
+        passive
+      });
+      this.delayEvents.current.push({ type, event, passive });
+    }
+  };
+
+  removeEvents = () => {
+    if (this.elementRef.current) {
+      this.delayEvents.current.forEach(({ type, event, passive }) => {
+        this.elementRef.current.removeEventListener(type, event, {
+          passive
+        });
+      });
+      this.delayEvents.current = [];
+    }
+  };
+
+  //A reference to the timeout handler to be able to access it at any time.
+  timeoutRef: { current: null | TimeoutID, ... } = React.createRef();
+
+  /**
+   * Start the delayed counter to determine when a drag should start.
+   * @param  {Event} e          TouchStart/MouseDown event.
+   */
+  startDelayTimeout: Event => void = e => {
+    this.delayEvents.current = [];
+    if (!this.state.allowedToDrag) {
+      /**
+       * Register events to cancel the timeout handler if user releases the mouse or touch
+       */
+      this.addEvent("mouseup", this.resetDelayTimeout);
```

Comment:
Is this needed? We should be able to handle this `onMouseUp`/`onDragStop`

Reply:
There's no need for mouse events since this feature is only enabled in touch devices.